test/suite/wttest.py

```diff
@@ -514,8 +514,19 @@ def extensionsConfig(self):
     def setUpConnectionOpen(self, home):
         self.home = home
         config = self.conn_config
+        
         if hasattr(config, '__call__'):
             config = self.conn_config()
+
+        # Collect all statistics for Python tests by default unless configured otherwise.
+        if not("statistics" in config):
+            if config != "":
```

Comment:
I don't think there is a need for this, if the config is "", then we can just have config = `,statistics=(all)` and that won't be an issue.